style/task-step.less

```diff
@@ -5,6 +5,12 @@
 @answer-hover-color: #e6f9fc;
 @answer-select-color: #def4f7; // darker than hover
 
+.task-step {
```

Comment:
`task-step` is the wrapping class.  It encloses the entire panel, and is used by the existing styles below.  Would you like it re-named to something more specific?